delta/sdk-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/views/indexing/IndexingStreamBehaviour.scala

```diff
@@ -188,49 +142,43 @@ object IndexingStreamBehaviour {
         // We just wait for the Stop message to come back from the shard
         def passivating(): Behaviors.Receive[IndexingViewCommand[V]] =
           Behaviors.receiveMessage[IndexingViewCommand[V]] {
-            case Stop =>
-              Behaviors.stopped
-            case _    =>
-              Behaviors.same
+            case Stop => Behaviors.stopped
+            case _    => Behaviors.same
           }
 
         // A stream is running for the current view
         def running(viewIndex: ViewIndex[V], switch: StreamSwitch): Behavior[IndexingViewCommand[V]] =
           Behaviors
             .receiveMessage[IndexingViewCommand[V]] {
-              case ViewRevision(rev) if rev > viewIndex.rev   =>
-                logger.debug(
-                  "'ViewRevision' event has been received and a newer revision exists for view {} in project {}",
-                  id,
-                  project
-                )
-                runStream(Some((viewIndex, switch)))
+              case ViewRevision(rev) if rev > viewIndex.rev =>
+                logger.debug("'ViewRevision' event has been received and a newer revision exists for view {} in project {}", id, project)
+                runStream(Some(Started(viewIndex, switch)), ProgressStrategy.Continue)
```

Comment:
A dedicated `ProgressStrategy` when a view is updated may make things easier ?

Reply:
I don't want to make things overly complicated. They are already a bit complicated. And I don't see a need for that distinction

Reply:
The new progress strategy could carry the information of the old view to stop and we could get rid of this option.
But it is just a thought, maybe it is not worth it.

Reply:
On the current state of things it does not seem very useful. We will see how it evolves with composite views, but I don't expect the need for anything more complicated.